README.md

```diff
@@ -236,10 +236,16 @@ CFLAGS to disable the compilation optimization parameters:
 ```shell
 $ export CFLAGS="-g -O0"
 $ ./build.sh
-$ ./configure
+$ ./configure --enable-assertions=yes
```

Comment:
I think we shouldn't suggest users to compile the library with `--enable-assertions=yes`. Seems like the default value is  `no` (based on [this](https://github.com/owasp-modsecurity/ModSecurity/pull/3207/files#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810R260) line), which is good.

I would rather call users **and** developers attention about the difference, and suggest developers to use `--enable-assertions=yes` - but only the developers.